EFAuditer/bin/Debug/netcoreapp2.2/EFAuditer.deps.json

```diff
@@ -0,0 +1,982 @@
+{

```

Comment:
I think we want to .gitignore this folder. Perhaps applying a **/bin/**/*  rule cross the codebase? :)